addons/mail/models/mail_followers.py

```diff
@@ -203,11 +204,12 @@ def _get_subscription_data(self, doc_data, pids, cids, include_pshare=False):
             where_clause += "AND (%s)" % " OR ".join(sub_where)
 
         query = """
-SELECT fol.id, fol.res_id, fol.partner_id, fol.channel_id, array_agg(subtype.id)%s
+SELECT fol.id, fol.res_id, fol.partner_id, array_agg(distinct u.id), fol.channel_id, array_agg(subtype.id)%s
```

Comment:
Why distinct ?

Reply:
I haven't pushed the correct commit, the correct version does a group by, the distinct was just a way to filter out duplicates while not changing the python code on my tests :P 


I have updated the code, with your remarks :)